arviz/plots/densityplot.py

```diff
@@ -57,6 +58,8 @@ def plot_density(
         List of variables to plot.  If multiple datasets are supplied and var_names is not None,
         will print the same set of variables for each dataset.  Defaults to None, which results in
         all the variables being plotted.
+    combine_dims : Optional[List[str]]
```

Comment:
This style is mypy like, and should be used only in type hints (that is in the code itself, not in docstrings). We initially had not set a convention on this so as you have seen each docstring has it's own style. We are progressively moving to type hints where they should be and numpydoc format for docstrings. In this particular case, I would use `set of str, optional` or `set_like of str, optional`

Reply:
This should be resolved! Please let us know if we happened to miss something. Thanks for the suggestion!